pandas/tests/indexes/test_base.py

```diff
@@ -878,24 +845,23 @@ def test_map_with_tuples(self):
         expected = MultiIndex.from_tuples([(i, i == 1) for i in idx])
         tm.assert_index_equal(result, expected)
 
+    def test_map_with_tuples_mi(self):
         # Test that returning a single object from a MultiIndex
         #   returns an Index.
         first_level = ['foo', 'bar', 'baz']
         multi_index = MultiIndex.from_tuples(lzip(first_level, [1, 2, 3]))
         reduced_index = multi_index.map(lambda x: x[0])
         tm.assert_index_equal(reduced_index, Index(first_level))
 
-    def test_map_tseries_indices_return_index(self):
-        date_index = tm.makeDateIndex(10)
+    @pytest.mark.parametrize("attr", [
+        'makeDateIndex', 'makePeriodIndex', 'makeTimedeltaIndex'])
```

Comment:
Will most likely replace with a fixture in the future, but made sense to parametrize for now